frontend/packages/knative-plugin/src/components/overview/KnativeResourceOverviewPage.tsx

```diff
@@ -45,27 +23,29 @@ const tabs = [
   },
 ];
 
-export const KnativeOverviewPage: React.ComponentType<
-  KnativeResourceOverviewPageProps
-> = getKsResourceModel(
-  ({ item, knativeModels, kindsInFlight }: KnativeResourceOverviewPageProps) => {
-    if (kindsInFlight) {
-      return !knativeModels ? null : <LoadingBox />;
-    }
-    const apiInfo = groupVersionFor(item.obj.apiVersion);
-    const resourceModel = knativeModels.find(
-      (model) =>
-        model.kind === item.obj.kind &&
-        model.apiGroup === apiInfo.group &&
-        model.apiVersion === apiInfo.version,
-    );
-    return (
-      <ResourceOverviewDetails
-        item={item}
-        kindObj={resourceModel}
-        menuActions={[...Kebab.getExtensionsActionsForKind(resourceModel), ...Kebab.factory.common]}
-        tabs={tabs}
-      />
-    );
-  },
-);
+export const KnativeOverviewPage: React.ComponentType<KnativeResourceOverviewPageProps> = ({
```

Comment:
filename `nativeResourceOverviewPage.tsx` is not matching with name of the component `KnativeOverviewPage` and even props `KnativeResourceOverviewPageProps`

Reply:
Good catch, I didn't see that.

Reply:
updated the component name and updated the imports.